commands/Moderation/kick.js

```diff
@@ -7,18 +7,21 @@ module.exports = class extends Command {
 			permissionLevel: 6,
 			requiredPermissions: ['KICK_MEMBERS'],
 			runIn: ['text'],
-
 			description: 'Kicks a mentioned user. Currently does not require reason (no mod-log).',
-			usage: '<member:member> [reason:string] [...]',
+			usage: '<member:user> [reason:string] [...]',
```

Comment:
member arg should use the member argument as kick requires the member to be present in the server.